pkg/fanal/analyzer/pkg/dpkg/copyright.go

```diff
@@ -130,3 +141,18 @@ func (a dpkgLicenseAnalyzer) Type() analyzer.Type {
 func (a dpkgLicenseAnalyzer) Version() int {
 	return dpkgLicenseAnalyzerVersion
 }
+
+// examples:
+// 'GPL-1+,GPL-2', 'GPL-1+, GPL-2'.
+// 'GPL-1+ or Artistic or Artistic-dist', 'LGPLv3+_or_GPLv2+', 'LGPL-2.1+    or     BSD-3-clause'
+// 'BSD-3-CLAUSE and GPL-2', 'GPL-1+ or Artistic, and BSD-4-clause-POWERDOG'
+// splitLicenses returns a list of licenses if the string contains multiple licenses
+func splitLicenses(line string) []string {
+	re := regexp.MustCompile("(,?[_ ]+or[_ ]+)|(,?[_ ]+and[_ ])|(,[ ]*)")
```

Comment:
`GPL 2.0` still doesn't work, right? And how about `GPL-1+ GPL-2`?

Reply:
oh.. i didn't add `GPL 2.0` to normalize map. 
Added that.

about `GPL-1+ GPL-2`:
I can't split licenses like `GPL-1+ GPL-2` because there are licenses like `GPL-2+ with Autoconf exception`
```
root@925d839bd202:/# cat /usr/share/doc/libnettle7/copyright | grep GPL-2+
License: LGPL-2+
License: GPL-2+
License: GPL-2+ with Autoconf exception
License: GPL-2+

```

Reply:
It's a mess...

Reply:
Can we define this regexp outside of the function?

Reply:
yes, of course.
Done